tests/python/pants_test/bin/test_loader_integration.py

```diff
@@ -20,6 +24,16 @@ def test_invalid_locale(self):
                                                             bypass_env: '1'})
     self.assert_success(pants_run)
 
+  @skip_unless_any_pythons_present(PY_26, PY_34, PY_35)
+  def test_invalid_interpreter(self):
```

Comment:
Bummer. I realized this test won't work in CI because we set `RUN_PANTS_FROM_PEX=1`, so the line `self.run_pants(command=['help'], extra_env={'PY': "python{}".format(invalid_versions_present[0])})` will run the PEX with the interpreter being used.

Even if we could figure out how to get the PEX to be ran with Py26, Py34, or Py35, we wouldn't want to because it would start bootstrapping a new venv folder and slow down CI dramatically.

Thoughts on ways around this, or should I kill the test?

Reply:
One potential workaround to this test failing in CI: 
- Patch `PantsLoader._is_supported_interpreter()` to always return `True`, meaning the exception will raise no matter which interpreter we're using. This would allow us to verify we do in fact we do in fact fail Pants and print a nice message. 
- We would not be testing `PantsLoader._is_supported_interpreter()`, though. Will write a unit test for this.

Will submit an update soon - let me know if you can think of a better design.